### ISSUE TITLE
Test failure EntryLoaderInitializationTest.testEagerEntryLoaderInitializesValues_withExpirationTime

### ISSUE THREAD

**Issue by petrpleshachkov** [hazelcast/hazelcast#15520]
http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/2487/

```
Error Message
expected:<val0> but was:<null>
Stacktrace
java.lang.AssertionError: expected:<val0> but was:<null>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.hazelcast.map.impl.mapstore.EntryLoaderInitializationTest.testEagerEntryLoaderInitializesValues_withExpirationTime(EntryLoaderInitializationTest.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

```

**Comment by devozerov** [comment]
The test fails because 5s TTL is not enough to load 100 entries. I tried to make test more reliable by switching from absolute expire time to relative TTL, but then the test started failing in another place because now it is not easy to determine when the entry is expired.

In the end, I simply decreased the number of entries being loaded so that it fits the 5s timeframe. It doesn't make the test 100% reliable, but at the very least I didn't observe a single failure after numerous runs.
